app/styleguide/wskComponentHandler.js

```diff
@@ -75,6 +75,11 @@ var componentHandler = (function() {
       var registeredClass = findRegisteredClass_(jsClass);
       if (registeredClass) {
         createdComponents_.push(new registeredClass.classConstructor(element));
+        // Call any callbacks the user has registered with this component type.
+        var length = registeredClass.callbacks.length;
+        for (var n = 0; n < length; n++) {
+          registeredClass.callbacks[n](element);
+        }
```

Comment:
No need for an anonymous function even:
```
registeredClass.callbacks.forEach(callback);
```
The imperative loop is definitely not going to gain us any performance and is bug-prone and introduces unnecessary mutability.